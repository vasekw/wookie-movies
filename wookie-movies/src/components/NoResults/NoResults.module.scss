.noResults {
  padding: 30px;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  gap: 20px
}

.noResultsTitle {
  color: white;
  font-weight: bold;
  font-size: 24px;
  text-align: center;
}