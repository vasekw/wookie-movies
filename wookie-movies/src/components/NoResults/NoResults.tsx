import React from "react";
import styles from "./NoResults.module.scss";
import Image from "next/image";

import { Fragment_Mono } from "next/font/google";
import classNames from "classnames";

export const fragmentMono = Fragment_Mono({
  subsets: ["latin"],
  display: "swap",
  weight: "400",
});

const NoResults: React.FC = () => {
  return (
    <div className={styles.noResults}>
      <div
        className={classNames(fragmentMono.className, styles.noResultsTitle)}
      >
        We searched Earth to no avail.
      </div>
      <Image src="/images/earth.png" alt="earth" width={250} height={250} />
      <div
        className={classNames(fragmentMono.className, styles.noResultsTitle)}
      >
        Try searching for something else.
      </div>
    </div>
  );
};

export default NoResults;
