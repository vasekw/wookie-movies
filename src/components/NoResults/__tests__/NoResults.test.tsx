import { render } from "@testing-library/react";
import NoResults from "../NoResults";

it("renders NoResults unchanged", () => {
  const { container } = render(<NoResults />);
  expect(container).toMatchSnapshot();
});
