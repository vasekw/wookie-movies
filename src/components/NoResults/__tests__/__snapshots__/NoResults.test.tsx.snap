// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NoResults unchanged 1`] = `
<div>
  <div
    class="noResults"
  >
    <div
      class="className noResultsTitle"
    >
      We searched Earth to no avail.
    </div>
    <img
      alt="earth"
      data-nimg="1"
      decoding="async"
      height="250"
      loading="lazy"
      src="/_next/image?url=%2Fimages%2Fearth.png&w=640&q=75"
      srcset="/_next/image?url=%2Fimages%2Fearth.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fearth.png&w=640&q=75 2x"
      style="color: transparent;"
      width="250"
    />
    <div
      class="className noResultsTitle"
    >
      Try searching for something else.
    </div>
  </div>
</div>
`;
